### ISSUE TITLE
Client Allocation Request for: Chengdu 23mofang Biotechnology Co., Ltd.

### ISSUE THREAD

**Issue by mftest2021** [filecoin-project/filecoin-plus-client-onboarding#403]
---

# Client Allocation Questions 

## Core Information
- Name: Chengdu 23mofang Biotechnology Co., Ltd.
- Website / Social Media: https://www.23mofang.com/
- Region: North America
- DataCap Requested: 30TiB
- Addresses to be Notarized: f1paantyyxvw562mpaqe45dnjhxe6pypectod7rya
- Notary Requested: ann-shin

**Comment by ann-shin** [comment]
Hi, nice to meet you. Thanks for your request. Can you kindly add more information here.
1.Are you representing for yourself or your company？
2.If you are representing for your company, Please give the following informations, duration of etablishment, number of employees, business scope, users volume, number of social media followers, data size to be stored, products or projects, and so on. If you are representing for yourself , please introduce yourself,such as number of social media followers, filecoin activities partipation with links, data size to be stored. Provide links to social media/news mentions.
3.The history working on IPFS / Filecoin, e.g. notable contributions to the community, type of engagement shown over time, etc.
Type, size, and source of the data you will be store on Filecoin,e.g. public data or private data, etc. Is there any sensitive or illegal content?
4.How do you intend to store the data on Filecoin, e.g. redundancy, location/miner preferences, etc
5.Have you already apply for DataCap to another notary, and whether it is accepted or rejected. Is there any reliable DataCap left?
6.Would you like to post real transactions on Github?

**Comment by mftest2021** [comment]
Hi, thanks for your reply. Please see our information below.
 
1.Are you representing for yourself or your company？

I'm representing our company, Chengdu 23mofang Biotechnology Co., Ltd.
 
2.If you are representing for your company, Please give the following informations, duration of establishment, number of employees, business scope, users volume, number of social media followers, data size to be stored, products or projects, and so on. If you are representing for yourself , please introduce yourself,such as number of social media followers, filecoin activities partipation with links, data size to be stored. Provide links to social media/news mentions.

Chengdu 23mofang Biotechnology Co., Ltd. was established in March 2015 with nearly 200 employees. We provide genetic testing services for Chinese people and carrying out life data analysis business. Relying on the Internet + genetic testing, our company builds a big database of life. We take the "digitizing life to benefit everyone" as our mission to build a trustworthy technology enterprise. The name, 23mofang (Mofang means Rubik's Cube in English ), derives from the fact that humans have 23 pairs of chromosomes, and "mofang" expresses polymorphism in genes. We now have about one million users and our data is more than 20PiB.
 
According to the "Analysis of China's Genetic Testing Industry 2018" released by Analysys（a third-party data institution in China）, 23mofang ranked first in the consumer-level genetic testing industry. At the same time, 23mofang was listed in the "50 Most Innovative Enterprises in China" released by Forbes China, and was evaluated by Forbes China as "gene testing innovator".
 
23mofang has accumulated 6 rounds of financing from investors including SoftBank China, Hanwang Technology, Jingwei China, and CD Capital, etc. It is the enterprise with the largest financing scale in the domestic consumption-level genetic testing industry.
 
We have a self-established gene testing center, which has been certified by the former National Health and Family Planning Commission and conforms to the standards of medical clinical testing institute. 23mofang also cooperates with the Ministry of Education's Key Laboratory of Modern Anthropology of Fudan University, Medical Genetics Center of the First Affiliated Hospital of Army Military Medical University, Kunming Institute of Zoology . Cas and other institutions to carry out scientific research.
 
Wechat official account: 23魔方（more than one million followers）
 
3.The history working on IPFS / Filecoin, e.g. notable contributions to the community, type of engagement shown over time, etc.
Type, size, and source of the data you will be store on Filecoin,e.g. public data or private data, etc. Is there any sensitive or illegal content?

We don't have any notable contributions to the community yet.  We want to store about 30TiB at this time for a small scale test and all these data are from genetic test samples, analysis results and genetic source data. They are all private data and can not be public with no sensitive/illegal content.
 
4.How do you intend to store the data on Filecoin, e.g. redundancy, location/miner preferences, etc

We want to store data in China with 3 copies. We prefer those miners with active working, high ranking, low cost.
 
5.Have you already apply for DataCap to another notary, and whether it is accepted or rejected. Is there any reliable DataCap left?

Yes, we have apply to another one notary before this application but its processing is slow.
 
6.Would you like to post real transactions on Github?

Yes, of course.

**Comment by ann-shin** [comment]
Your organization is not in North America。
According to our allocation strategy(https://github.com/filecoin-project/notary-governance/issues/27)，please wait for further notice after  7 days.

**Comment by mftest2021** [comment]
OK, @XnMatrixSV if more information is required after 7 days, please tell us at any time.

Besides, we'd like to say that 23 Mofang ranks first in the consumer genetic testing industry and is committed to building the largest life database in China. By collecting saliva samples, users can perform ancestry analysis, genetic risk, genetic traits, genetic variation carrying, nutritional needs, drug response, sports health, skin management and other hundreds of genetic tests. These real data, including genetic test samples, analysis results and genetic source data, constitute a huge gene database.

In short, through genetic testing, 23 Mofang helps people better understand themselves and targeted disease prevention, which is conducive to improving the human health index. At the same time,  the huge testing and analysis data of 23 Mofang has also greatly promoted the development of medical research and genetic science research.
